addons/mail/static/src/core/common/thread.js

```diff
@@ -273,7 +283,7 @@ export class Thread extends Component {
         });
         onWillDestroy(() => stopOnChange());
         const saveScroll = () => {
-            this.props.thread.scrollTop =
+            toRaw(this.props.thread).scrollTop =
```

Comment:
Can/should we do this fix in stable too?

Reply:
We should definitely do this in stable as it makes scrolling the message list jittery. I'd rather back porting it not to block this PR though.